django-stubs/contrib/gis/geos/geometry.pyi

```diff
@@ -82,7 +82,7 @@ class GEOSGeometryBase(GEOSBase):
     def json(self) -> str: ...
     geojson: str
     @property
-    def wkb(self) -> bytes: ...
+    def wkb(self) -> memoryview: ...
     @property
     def ewkb(self) -> bytes: ...
```

Comment:
Looks like `ewkb` in Django has similar code and returns `memoryview` as well?

Have a look at other `bytes` members in these classes: `hexewkb`, `hex`?

Reply:
It looks like ewkb needs to be fixed ...

```python
>>> ds = gdal.DataSource("sample.shp")

# GEOSGeometry
>>> type(ds[0][0].geom.geos.ewkb)
memoryview
>>> type(ds[0][0].geom.geos.hexewkb)
bytes
>>> type(ds[0][0].geom.geos.hex)
bytes

# OGRGeometry
>>> type(ds[0][0].geom.wkb)
memoryview
>>> type(ds[0][0].geom.hex)
bytes
```